core/webviz/_json_page_element.py

```diff
@@ -12,14 +13,16 @@
 _json_store_init.name = 'json_store_init.js'
 
 
-def default(o):
-    if isinstance(o, numpy.int64):
-        return int(o)
-    raise TypeError
+def default(data):
+    if isinstance(data, numpy.int64):
+        return int(data)
+    if isinstance(data, (datetime.datetime, datetime.date)):
+        return data.__str__()
+    raise TypeError('Given data can not be converted to a json format')
```

Comment:
The `TypeError` should probably contain a description of what type was attempted to convert.